homework_1/02_ProgressiveTax/taxes/taxes.go

```diff
@@ -0,0 +1,60 @@
+package taxes
+
+import "github.com/pkg/errors"
+
+type taxBrackets struct {
+	brackets []float32
+	taxes    []float32
+}
+
+func checkTaxBrackets(brackets []float32, taxes []float32) error {
+	if len(brackets) == 0 || brackets[0] != 0 {
+		return errors.New("First bracket is not zero.")
```

Comment:
Error message shouldn't have punctuation nor capitalisation. (i.e. this should be `"first bracket is not zero"`).
This is due to a usual pattern in Go where each caller in the chain will wrap the error it received with some additional information. Wrapping is usually done by prefixing "additional information: " to the error and if every error (and wrapper) had punctuation the final result would look quite weird.